docs/content/misc/contribute.ngdoc

```diff
@@ -61,7 +61,8 @@ To build AngularJS, you clone the source code repository and use Grunt to genera
 minified AngularJS files:
 
 ```shell
-# Clone your Github repository:
+# Clone your Github repository(Note: Do not go with `sudo` user when cloning into this repository 
```

Comment:
I don't see it as particularly helpful to tell the user not to use sudo even though this point doesn't mention sudo at all.